sdk/core/core-http/review/core-http.api.md

```diff
@@ -233,7 +233,11 @@ export interface EnumMapperType {
     // (undocumented)
     allowedValues: any[];
     // (undocumented)
+    deSerializingMapper?: Function;
```

Comment:
What do you think about passing in an object with key value pairs that can be used to serialize/deserialize instead of 2 functions.

something like

```typescript
enumMap: {
   nameOne: "one",
   nameTwo: "two",
   nameThree: "three"
}
```

And later on 

```typescript
// Serialization
const value = mapper.enumMap[value];

// Deserialization
const value = Object.keys(mapper.enumMap).find(k =>mapper.enumMap[k] === value );
```

With this the generator doesn't need to generate the translate function, just the mapping object and pass it to core-http. 

What do you think?

Reply:
Looks great. Let me try the code to see if it will work and then update the code.

Reply:
Updated the code to use enumMaps instead of functions.